schema.asn

```diff
@@ -61,6 +61,32 @@ Configuration-Structure-Schema DEFINITIONS AUTOMATIC TAGS ::= BEGIN
       single SEQUENCE OF INTEGER OPTIONAL,
       range SEQUENCE OF Range OPTIONAL
    }
+
+   CustomOctetString ::= SEQUENCE {
+      -- Ideally we'd build something based on information object classes.
+      -- asn1tools doesn't support those, however. Some day available tools
+      -- might improve. When that day comes we can stop using this type and
+      -- use something better.
+
+      -- Vendor-specific ID for this data.
+      id INTEGER,
+      data OCTET STRING
+   }
+
+   Custom ::= SEQUENCE {
+      -- Bits 6:0 of the designer/manufacturer’s Identification Code as assigned
+      -- by JEDEC Standard JEP106.
+      jedecLow INTEGER (0..127),
+      -- The modulo 32 count of the number of continuation characters (0x7f) in
+      -- that same Identification Code.
+      jedecContinuation INTEGER (0..31),
```

Comment:
Suppose this ID  is the same as the mvendorid of the hart, right?
So firmware can use this ID to validate if the custom data exactly for this hart.

Reply:
Good point. I should phrase this to use the same kind of definition/language as we have for mvendorid.